### PR TITLE
move config.ini to config.example to make upgrades easier.

### PR DESCRIPTION
**Pull request by stlorenz**
I have renamed the example config to `config.example` and updated the doc accordingly. 
With that change, an update does not overwrite the existing `config.ini`

Update .gitignore and README.md files

### CODE REVIEW

**Comment by dsteinkopf** [comment]
I like this!

**Comment by henne49** [comment]
@0x7878 we still have the Versioning failing, can you check in the github action

**Comment by 0x7878** [comment]
> @0x7878 we still have the Versioning failing, can you check in the github action

> See #164 but either we have to configure a secured way to this or we have to drop this feature.

**Comment by 0x7878** [comment]
@stlorenz don't worry. It's just us. We have to improve our GitHub Actions. That's all. We need more time to get it done.

**Comment by 0x7878** [comment]
If you push something to this repo, the error should go away and we can merge this.

**Comment by dsteinkopf** [comment]
Look good to me.

File: README.md [hunk 1]
```diff
@@ -65,6 +67,7 @@ chmod a+x /data/dbus-opendtu/*.sh
 ⚠️**Edit the following configuration file according to your needs before proceeding**⚠️ see [Configuration](#configuration) for details.
 
 ```bash
+mv /data/dbus-opendtu/config.example /data/ddbus-opendtu/config.ini
```
Comment (dsteinkopf) [inline README.md:70]: This should be a "cp".

**Review by dsteinkopf — CHANGES_REQUESTED** [review]

**Review by henne49 — APPROVED** [review]
this looks very good, ready to merge

**Review by dsteinkopf — APPROVED** [review]

**Pull request merged**